SPEC.md

```diff
@@ -85,7 +91,7 @@ Success is indicated by a return code of zero and the following JSON printed to
 
 ```
 {
-  "cniVersion": "0.2.0",
+  "cniVersion": "0.3.0",
```

Comment:
If we care about people building from master, what's the point of having numbered releases then?  Our releases are currently match our Spec version, so we would either have to uncouple these two.  Plus we'd have to bump the Spec version every time a PR made an incompatible change.  We couldn't bundle up multiple spec changes spread across a couple commits into one real release, which will lead to Spec version inflation.